app/models/user.rb

```diff
@@ -35,7 +35,5 @@ class User < ApplicationRecord
   include DeviseTokenAuth::Concerns::User
 
   enum gender: { female: 0, male: 1, fluid: 2 }
-
-  validates :email, presence: true, uniqueness: true
```

Comment:
Because I had a problem with the rubocop, When I used the email restriction it was required, rubocop gave me an error because the validation was being done by the provider. was to change this or change the devise restrictions